### PR TITLE
Price Context tracking

### PR DESCRIPTION
**Pull request by lilyfromseattle**
_-When users are evaluating the price of an artwork, I want to be able to track the following price summaries interactions:__

Scrolling down to the price summaries (with 1x1 pixel firing)

Hover over bars in the chart

Clicks into range -> view collect page

Clicks on question mark icon

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 398396c58369deb54e714ab4e84aee550e61af96

https://deploy-preview-2307--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/PricingContext.tsx [hunk 1]
```diff
@@ -38,78 +41,115 @@ function PricingContext({ artwork }: PricingContextProps) {
...
+  }
+
+  @track({
+    action_type: Schema.ActionType.Hover,
+    flow: Schema.Flow.ArtworkPriceContext,
+    subject: Schema.Subject.HistogramBar,
+    type: Schema.Type.Chart,
+  })
+  barchartHover() {
+    // I'm just for tracking!
```
Comment (sweir27) [inline src/Apps/Artwork/Components/PricingContext.tsx]: 😊

File: src/Apps/Artwork/Components/PricingContext.tsx [hunk 2]
```diff
@@ -38,78 +41,115 @@ function PricingContext({ artwork }: PricingContextProps) {
...
+                priceCents >= bin.minPriceCents &&
+                priceCents < bin.maxPriceCents
+
+              const binValue =
+                artworkFallsInThisBin && bin.numArtworks === 0
+                  ? 1
+                  : bin.numArtworks
+              const labelSuffix = binValue === 1 ? " work" : " works"
+              return {
+                value: bin.numArtworks,
```
Comment (sweir27) [inline src/Apps/Artwork/Components/PricingContext.tsx]: Is this a regression?

File: src/Apps/Artwork/Components/__tests__/PricingContext.test.tsx [hunk 3]
```diff
@@ -162,4 +169,57 @@ describe("PricingContext", () => {
...
+      expect(dispatch).toBeCalledWith({
+        action_type: "Impression",
+        context_module: "Price Context",
+        subject: "Histogram Bar",
+        type: "Chart",
+        flow: "Artwork Price Context",
+      })
+    })
+
+    it("tracks clicks on histogram bars", () => {
```
Comment (sweir27) [inline src/Apps/Artwork/Components/__tests__/PricingContext.test.tsx]: Wowee these tests are awesome!!!!

File: src/Apps/Artwork/Components/PricingContext.tsx (hunk 2 above)
Comment (lilyfromseattle) [reply src/Apps/Artwork/Components/PricingContext.tsx]: Idk I had really bad merge conflict but I thought I fixed it

**Comment by sweir27** [comment]
Looks good! I'm not sure what that generated file is all about-- but `MarketInsightsArtistPage_artist` doesn't show up anywhere in the codebase so I think it's OK to remove.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.0.2 :rocket:

**Pull request merged**